pkg/config/ao.go

```diff
@@ -156,10 +153,18 @@ func (ao *AOConfig) AddMultipleClusterConfig() {
 		"prod-relay": {
 			Type: "ocp3",
 		},
-		newCluster: {
+	}
+
+	ocp4Clusters := []string{"prod01", "test01", "utv04"}
+	for _, cluster := range ocp4Clusters {
+		ao.AvailableClusters = append(ao.AvailableClusters, cluster)
+		ao.PreferredAPIClusters = append([]string{cluster}, ao.PreferredAPIClusters...)
```

Comment:
Hvorfor legges ocp4Clusters til PreferredAPIClusters uten videre?

Reply:
Alle kall som opererer på AuroraConfig går kun mot en Boober. Tror det er bare deploy som vil gå mot flere basert på cluster-verdien i AuroraConfig. Så preferredAPIClusters brukes for å avgjør hvilken boober som skal bli brukt for operasjoner på AuroraConfig.
Det er med andre ord ikke noen god grunn til hvorfor prod ikke har vært med i denne listen tidligere. Gjør det mer robust med flere.

AO burde vært like smart som på deploy, hvor den sender AuroraConfig til riktig cluster (fordi det kan være noen features som er slått av i valgt cluster).